src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp

```diff
@@ -104,6 +104,17 @@ DECLARE_CONFIG_KEY(FORCE_DISABLE_CACHE);
  */
 DECLARE_CONFIG_KEY(CONFIG_DEVICE_ID);
 
+/**
+ * @brief enable hyper thread
+ */
+DECLARE_CONFIG_KEY(ENABLE_HYPER_THREAD);
+
+/**
+ * @brief generic boolean values
+ */
+DECLARE_CONFIG_VALUE(YES);
+DECLARE_CONFIG_VALUE(NO);
+
```

Comment:
I suppose config value not needed to re-declare?

Reply:
done